src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -208,6 +208,12 @@ public OpenIdConnectHandler(HttpClient backchannel, HtmlEncoder htmlEncoder)
 
             message.State = Options.StateDataFormat.Protect(properties);
 
+            if (string.IsNullOrEmpty(message.IssuerAddress))
+            {
+                throw new InvalidOperationException(
+                    $"Cannot redirect to the end session endpoint, the configuration may be missing or invalid: '{message.IssuerAddress}'");
+            }
```

Comment:
Messages and issuers are generic terms in the object model, if you look up a few lines you will see issuer assigned from an end session endpoint.